app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,6 @@
     <string name="ratingLabel">Rating</string>
     <string name="releaseDateLabel">Release Date</string>
     <string name="genresLabel">Genres</string>
+    <string name="emptyDatabaseText">I have no data to show</string>
+    <string name="genericContentDescription">NULL</string>
```

Comment:
Just use null in XML instead of creating generic content description to bypass warnings 😄